src/Components/CookieNotice.js

```diff
@@ -18,6 +18,7 @@ import {
   validateMaxWidth,
   validateCookieExpiration,
   validateDarkTheme,
+  validateCookieName,
```

Comment:
And this line just below `validateCookieExpiration` import ! (I like ordering lol)